policy/lib/tekton/task.rego

```diff
@@ -21,16 +21,12 @@ latest_required_default_tasks := ectime.newest(data["required-tasks"])
 current_required_default_tasks := ectime.most_current(data["required-tasks"])
 
 # retrieve the tasks from the required task list
-get_tasks(required_tasks) := tasks if {
-	tasks := [task | 
-		some task in required_tasks.tasks
-	]
-}
+required_tasks(required_tasks) := [task |
+	some task in required_tasks.tasks
+]
```

Comment:
It seems more significant than just lint fixes. How about squashing this one in with the previous commit?

Reply:
yeah, the regal linter caught this. I'll squash them. I left the changes as their own commit to show they weren't part of the main change.